### PR TITLE
ENH: Add ActiveColor property for markup display

### PR DESCRIPTION
**Pull request by Sunderlandkyl**
Allows the color shown when the user hovers over a markup component to be changed.

### CODE REVIEW

File: Modules/Loadable/Markups/MRML/vtkMRMLMarkupsDisplayNode.h [hunk 1]
```diff
@@ -397,6 +397,11 @@ class  VTK_SLICER_MARKUPS_MODULE_MRML_EXPORT vtkMRMLMarkupsDisplayNode : public
   /// This function should always return a valid vtkTextProperty pointer.
   vtkGetObjectMacro(TextProperty, vtkTextProperty);
 
+  /// Set the active color of the markup.
```
Comment (lassoan) [inline Modules/Loadable/Markups/MRML/vtkMRMLMarkupsDisplayNode.h:400]:
```suggestion
  /// Set the active color of the markup. This color is used when the mouse pointer hovers over a markup.
```

File: Modules/Loadable/Markups/MRML/vtkMRMLMarkupsDisplayNode.h [hunk 2]
```diff
@@ -397,6 +397,11 @@ class  VTK_SLICER_MARKUPS_MODULE_MRML_EXPORT vtkMRMLMarkupsDisplayNode : public
...
+  vtkSetVector3Macro(ActiveColor, double);
+  /// Get the active color of the markup.
```
Comment (lassoan) [inline Modules/Loadable/Markups/MRML/vtkMRMLMarkupsDisplayNode.h:402]:
```suggestion
  /// Get the active color of the markup. This color is used when the mouse pointer hovers over a markup.
```

**Review by lassoan — DISMISSED** [review]
Looks good, thank you. I've just added two small suggestions for improving comments.

**Comment by Sunderlandkyl** [comment]
Thanks @lassoan, I've added the additional comment.

**Review by lassoan — APPROVED** [review]
Thank you!

**Pull request merged**